.github/workflows/main.yml

```diff
@@ -10,8 +10,6 @@ jobs:
       fail-fast: false
       matrix:
         include:
-        - python: "3.6"
-          docutils: du14
```

Comment:
Note: Ubuntu-20.04 distributes docutils-0.16. So it will be the new supported base version. I'll work on it after merging this.
https://packages.ubuntu.com/focal/python3-docutils

Reply:
Would it be possible to increase the minimum requirement to Docutils 0.18? There are lots of patches in Sphinx for Docutils 0.17 or earlier, and it would be nice to remove those at the same time as we drop end-of-life Python versions.

As far as I can tell the current Sphinx version support policy only makes reference to Ubuntu for Python versions, not for Docutils.

A

Reply:
In my thought, it's simple to refer to Ubuntu LTS to decide the version of dependencies. I feel dropping dependency does not help our work. So -0 for your proposal

Reply:
> I feel dropping dependency does not help our work.

We could take a view to support every version of Python / Docutils perpeptually, and just accumulate a lot of patches. I personally find it harder to reason about the code when there are multiple layers of patches etc, especially ones that monkeypatch Docutils, hence looking to bump the version higher and remove code.

I'd argue we should look to make things simpler for ourselves as maintainers where possible - Ubuntu is a re-distributor and will cope with decisions that we make under their policies, but I don't think we should be somewhat arbitrarily bound by them when there are significant code simplifications from the bump to 0.18.

A